tests/test_llama_guard.py

```
@@ -52,6 +66,45 @@
+    for action_func in (llama_guard_check_input, llama_guard_check_output):
+        outcome = await action_func(
+            llm_task_manager=task_manager,
+            context=context,
+            llama_guard_llm=FakeLLMModel(responses=[llm_response]),
+        )
+
+        assert outcome == expected
+
+
+def test_llama_guard_output_action_metadata_is_registration_only():
+    assert set(getattr(llama_guard_check_output, "action_meta")) == {
```

Ruff (0.15.21)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)